wiquery-jquery-ui/src/test/java/org/wicketstuff/wiquery/ui/datepicker/DatePickerLanguageResourceReferenceTestCase.java

```diff
@@ -111,22 +110,4 @@ public void testRenamedLocales_JDK17OrLater()
 		// kz does not exist
 		assertNotEquals(new Locale("kk"), new Locale("kz"));
 	}
-
-	@Test
-	@EnabledForJreRange(max =JRE.JAVA_16)
```

Comment:
I think you could optimize the imports. Since the annotations are no more needed